src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -634,6 +667,132 @@ def get_request_status(
     return paginated
 
 
+@router.get(
+    PRIVACY_REQUESTS,
+    dependencies=[Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_READ])],
+    response_model=Page[
+        Union[
+            PrivacyRequestVerboseResponse,
+            PrivacyRequestResponse,
+        ]
+    ],
+    deprecated=True,
+)
+def get_request_status(
+    *,
+    db: Session = Depends(deps.get_db),
+    params: Params = Depends(),
+    request_id: Optional[str] = None,
+    identity: Optional[str] = None,
+    status: Optional[List[PrivacyRequestStatus]] = FastAPIQuery(
+        default=None
+    ),  # type:ignore
+    created_lt: Optional[datetime] = None,
+    created_gt: Optional[datetime] = None,
+    started_lt: Optional[datetime] = None,
+    started_gt: Optional[datetime] = None,
+    completed_lt: Optional[datetime] = None,
+    completed_gt: Optional[datetime] = None,
+    errored_lt: Optional[datetime] = None,
+    errored_gt: Optional[datetime] = None,
+    external_id: Optional[str] = None,
+    action_type: Optional[ActionType] = None,
+    verbose: Optional[bool] = False,
+    include_identities: Optional[bool] = False,
+    include_custom_privacy_request_fields: Optional[bool] = False,
+    download_csv: Optional[bool] = False,
+    sort_field: str = "created_at",
+    sort_direction: ColumnSort = ColumnSort.DESC,
+) -> Union[StreamingResponse, AbstractPage[PrivacyRequest]]:
+    """
+    **This endpoint is deprecated. Please use `POST /privacy-request/search`,
+    which uses body parameters instead of query parameters for filtering.**
+
+    Returns PrivacyRequest information. Supports a variety of optional query params.
+
+    To fetch a single privacy request, use the request_id query param `?request_id=`.
+    To see individual execution logs, use the verbose query param `?verbose=True`.
+    """
+
+    return _shared_privacy_request_search(
+        db=db,
+        params=params,
+        request_id=request_id,
+        identity=identity,
+        identities=None,
+        custom_privacy_request_fields=None,
```

Comment:
Could you call this out, why these two fields are null, just to make it more easily parseable how you're sharing this method between the old endpoint and the new endpoint -

Reply:
> Both the old and new versions of the privacy request search endpoints use this shared function.
    The `identities` and `custom_privacy_request_fields` parameters are only supported in the new version
    so they are both set to None here.